photon-api/src/main/scala/com/linkedin/photon/ml/projector/IndexMapProjectorRDD.scala

```diff
@@ -88,15 +88,44 @@ protected[ml] class IndexMapProjectorRDD private (indexMapProjectorRDD: RDD[(Str
    */
   override def projectCoefficientsRDD(
       modelsRDD: RDD[(String, GeneralizedLinearModel)]): RDD[(String, GeneralizedLinearModel)] =
+
+    // Left join the models to projectors for cases where we have a prior model but no new data (and hence no
+    // projectors)
+    modelsRDD
+      .leftOuterJoin(indexMapProjectorRDD)
+      .mapValues { case (model, projectorOpt) =>
+        projectorOpt.map { projector =>
+          val oldCoefficients = model.coefficients
+
+          model.updateCoefficients(
+            Coefficients(
+              projector.projectCoefficients(oldCoefficients.means),
+              oldCoefficients.variancesOption.map(projector.projectCoefficients)))
+        }.getOrElse(model)
+      }
+
+  /**
+   * Project a [[RDD]] of [[GeneralizedLinearModel]] [[Coefficients]] from the original space to the projected space.
+   *
+   * @param modelsRDD The input [[RDD]] of [[GeneralizedLinearModel]] with [[Coefficients]] in the original space
+   * @return The [[RDD]] of [[GeneralizedLinearModel]] with [[Coefficients]] in the projected space
+   */
+  override def transformCoefficientsRDD(
```

Comment:
Need a better name, so that it's immediately clear which function projects TO the new space and which one FROM the new space.

Reply:
Yep. Same as above.